File: Aplikacija/Frontend/Angular/src/app/Korisnik.ts
export class Korisnik
{
    KorisnickoIme:string;
    Email:string; 
    Grad:string;
    Kontakt:string;
    Ime:string;
    Naziv:string;
    Prezime:string;
    Lokacija:string;
    Slika:string;

    email:string;
    tip:string;
    korisnicko_ime:string;
    ime:string;
    naziv:string;
    id:number;
    imgPath:string;
    token:string;
    
    constructor()
    {
        this.KorisnickoIme = "";
        this.Lokacija = ""; 
        this.Prezime = "";   
        this.Kontakt = "";
        this.Email = "";
        this.email = "";
        this.Naziv = "";
        this.Grad = "";  
        this.Ime = "";    
        this.ime = "";
        this.tip ="";
        this.Slika="";
        this.korisnicko_ime="";
        this.imgPath="-1"
        this.token = "";
    }
}

File: Aplikacija/Backend/SW_APP/Models/Korisnik.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Models
{
    [Table("Korisnik")]
    public class Korisnik
    {
        [Key]
        public int ID { get; set; }
        
        
        public string lozinka { get; set; }
        public byte[] salt_value {get; set; }

        [MaxLength(30)]
        public string korisnicko_ime { get; set; }
        [MaxLength(60)]
        public string email { get; set; }
        
        [MaxLength(10)]
        public string tip { get; set; }  

        public string grad { get; set; }

        public string imgPath {get;set;}

        
    }
}
